src/openrct2-unity/Assets/Scripts/OpenRCT2/Map/PeepController.cs

```diff
@@ -67,8 +67,6 @@ protected override SpriteObject AddSprite(int index, ref Peep sprite)
         protected override SpriteObject UpdateSprite(int index, ref Peep sprite)
```

Comment:
You can remove this entire method as the override currently doesn't do anything extra anymore. :)

Reply:
Uhm you know you wrote that right? I do not not even know what it is supposed to do. xD

Reply:
I know haha, I added it for your method that updated PeepInformation. But since that has been refactored, this method becomes obsolete. :p